apisix/http/router/radixtree_host_uri.lua

```diff
@@ -94,7 +94,15 @@ local function create_radixtree_router(routes)
     host_router = nil
 
     for _, route in ipairs(routes or {}) do
+        local status = core.table.try_read_attr(route, "value", "status")
+        -- check the status
+        if status and status == 0 then
+            goto CONTINUE
```

Comment:
Do we really need "goto"?

Reply:
you mean like this?
```
        if status = 1 then
              push_host_router(route, host_routes, only_uri_routes)
        end
```

Reply:
yes

Reply:
and we need to deal with old data which not has status

Reply:
get